cms/envs/aws.py

```diff
@@ -71,10 +71,18 @@
 # Each worker should only fetch one message at a time
 CELERYD_PREFETCH_MULTIPLIER = 1
 
+#
 # Skip djcelery migrations, since we don't use the database as the broker
-SOUTH_MIGRATION_MODULES = {
+#
+# don't clobber any SOUTH_MIGRATION_MODULES configuration that might have
+# been set elsewhere, like in common.py
+#
+if not 'SOUTH_MIGRATION_MODULES' in vars() and not 'SOUTH_MIGRATION_MODULES' in globals():
+    SOUTH_MIGRATION_MODULES = {}
+
+SOUTH_MIGRATION_MODULES.update({
     'djcelery': 'ignore',
-}
+})
```

Comment:
We discovered that this change is needed when we deployed edx-notifications to McKinsey Academy. Just carrying this forward.